andela/bad_ass-ah-frontend
# #164489901 article reading time

**What does this PR do?**

This PR shows time it takes to read an article

**Description of Task to be completed?**

User fetch a particular article to see the time

**How should this be manually tested?**

After cloning the repo, run ```npm run dev``` to start the server

- Navigate to home page
- Click on an article for reading and see the time it takes to read it

**Any background context you want to provide?**

Previously, the application was not showing time it takes to read an article but now this PR does it

**What are the relevant pivotal tracker stories?**

[#164489901](https://www.pivotaltracker.com/story/show/164489901)

**Screenshots (if appropriate)**

<img width="1439" alt="Screen Shot 2019-06-16 at 17 35 14" src="https://user-images.githubusercontent.com/33277011/59566319-d1a3a880-905e-11e9-8016-ff6efa068473.png">
